Library/Homebrew/formulary.rb

```diff
@@ -27,18 +27,37 @@ def self.formula_class_get(path)
     cache.fetch(path)
   end
 
-  def self.load_formula(name, path, contents, namespace)
+  def self.load_formula(name, path, contents, namespace, flags:)
     raise "Formula loading disabled by HOMEBREW_DISABLE_LOAD_FORMULA!" if Homebrew::EnvConfig.disable_load_formula?
 
     mod = Module.new
     const_set(namespace, mod)
+
+    class_name = class_s(name)
+
     begin
-      mod.module_eval(contents, path)
+      if contents.respond_to?(:call)
```

Comment:
Why is this needed now and wasn't before?

Reply:
Leftover from debugging. Removed.